frame/composable-traits/src/rate_model.rs

```diff
@@ -47,28 +47,6 @@ pub type Ratio = FixedU128;
 /// accounts to length of year)
 pub const SECONDS_PER_YEAR: DurationSeconds = 365 * 24 * ONE_HOUR;
 
-/// utilization_ratio = total_borrows / (total_cash + total_borrows)
```

Comment:
assume copy paste

Reply:
yes, after removing it no compile-time complaints nor any tests broke.